src/shared/src/types/user-types.ts

```diff
@@ -39,6 +39,7 @@ export interface AuthenticatedUser {
   defaultTheme?: ThemeName;
   teamAsLeadId?: string;
   favoritedProjectsId?: number[];
+  changeRequestsToReviewId?: number[];
```

Comment:
way r these two optional they should just be empty lists

Reply:
yeah empty lists would p be better

Reply:
we do this in a bunch of places rn and idk why there might be a reason